workflows/ci-simple/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -0,0 +1,166 @@
+  # Use your docker credentials for the runtime namespace
+  # https://jamesdefabia.github.io/docs/user-guide/kubectl/kubectl_create_secret_docker-registry/
+  volumes:
+    - name: docker-config
+      secret:
+        items:
+          - key: .dockerconfigjson
+            path: config.json
+        secretName: docker-registry # this is the name of the docker-registry secret you created
+  entrypoint: ci-steps
+  templates:
+    - name: ci-steps
+      inputs:
+        parameters:
+          - name: REPO
+          - name: IMAGE_NAME
+          - name: TAG
+            default: latest
+          - name: GIT_REVISION
+            default: main         # use default value if this parameter is not provided
+          - name: GIT_BRANCH
+            default: main
+          - name: GIT_COMMIT_URL
+            default: ''
+          - name: GIT_COMMIT_MESSAGE
+            default: ''
+          - name: DOCKERFILE
+            default: Dockerfile
+          # Create git secret token and apply it to your cluster under the runtime namespace
+          # NOTE: The token should be in base64 format
+          # https://kubernetes.io/docs/concepts/configuration/secret/
+          # Create github.secret.yaml with the following content:
+          #   apiVersion: v1
+          #   kind: Secret
+          #   metadata:
+          #     name: github-token
+          #   type: Opaque
+          #   data:
+          #     token: <github token base64 here>
+          # Run kubectl -n <namespace> apply -f github.secret.yaml
+          - name: GIT_TOKEN_SECRET
+            default: github-token # the name of the git secret token you applied to your cluster
+          - name: CONTEXT
+            default: .
+          # Create registry-creds secret and apply it to your cluster under the runtime namespace
+          # You can use also use other creds such gcr/aws; please look at the template inputs for more info  https://github.com/codefresh-io/argo-hub/blob/main/workflows/codefresh-csdp/versions/0.0.6/docs/report-image-info.md
+          # Note that the token uses base64
+          # https://kubernetes.io/docs/concepts/configuration/secret/
+          # Create registry-creds.yaml with the following content:
+          #   apiVersion: v1
+          #   kind: Secret
+          #   metadata:
+          #     name: registry-creds
+          #   type: Opaque
+          #   data:
+          #     username: <REGISTRY_USERNAME base64 here>
+          #     password: <REGISTRY_PASSWORD base64 here>
+          #     domain: <REGISTRY_DOMAIN base64 here>
+          # Run kubectl -n <namespace> apply -f <registry-creds name>.yaml
+          - name: REGISTRY_CREDS
+            default: registry-creds  # the name of the registry-creds secret you created
+      dag:
+        tasks:
+          # https://github.com/codefresh-io/argo-hub/blob/main/workflows/kaniko/versions/0.0.1/docs/build.md
+          # https://codefresh.io/argohub/workflow-template/kaniko
```

Comment:
I agree with Itai that we should move a lot of the comments to the documentation, but I really like having the Argo Hub links before each templateRef. I think it makes it a lot easier and quicker to understand/use this workflow template, so I hope we keep that. We could probably remove the github link though - one link is simpler than 2.